distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/opensearch/OpenSearchTestCommons.java

```diff
@@ -29,27 +34,66 @@
 
     public static final String OPENSEARCH_FACTORY_PID = "OpenSearchSource";
 
-    public static Map<String, Object> getOpenSearchSourceProperties(String sourceId, String openSearchUrl, ServiceManager serviceManager) {
+    public static final AbstractIntegrationTest.DynamicPort HTTPS_PORT =
+            new AbstractIntegrationTest.DynamicPort("org.codice.ddf.system.httpsPort", 2);
+
+    public static final AbstractIntegrationTest.DynamicUrl SERVICE_ROOT =
+            new AbstractIntegrationTest.DynamicUrl(AbstractIntegrationTest.DynamicUrl.SECURE_ROOT,
+                    HTTPS_PORT,
+                    "/services");
+
+    public static final AbstractIntegrationTest.DynamicUrl REST_PATH =
+            new AbstractIntegrationTest.DynamicUrl(SERVICE_ROOT, "/catalog/");
+
+    public static final AbstractIntegrationTest.DynamicUrl OPENSEARCH_PATH =
```

Comment:
This constant and the ones above should only be set in the `AbstractIntegrationTest` class.